src/wacks4shop/shift4shop.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+from typing import Optional
+
+from wacksbywarby.models import Sale
+
+logger = logging.getLogger("shift4shop")
+
+
+class Shift4Shop:
+    def __init__(self, debug=False) -> None:
+        self.key = os.getenv("SHIFT4SHOP_PRIVATE_KEY")
+        self.shop_token = os.getenv("SHIFT4SHOP_SHOP_TOKEN")
+        self.debug = debug
+
+    def determine_sales(self, timestamp: Optional[str]) -> list[Sale]:
+        """
+        Query the Shift4Shop API for all orders since the given timestamp. Then transform these orders
+        into a list of Sale objects
+        """
+        # TODO
+        return []
+
+    def get_num_sales(self) -> int:
```

Comment:
I might have missed it but I didn't see an API to get the current total number of sales. Do we need still need this function though? With shift4shop you don't need to compare the current total number of sales with the old total number of sales. As long as the orders api returns orders with status new after the given timestamps, you know a sale has been made. If you do not get anything back then you know there is no sales.

Reply:
ah okay. in the past we appended a message at the end of each message saying how many total sales there have been (so we know when to have a board meeting!) I wonder if we can get it via that `countonly` param and just count all orders that are completed?

Reply:
oh great point. You're right we prob still want to know the total number of sales for that. That make sense. It would be the same call to the orders api with `countonly=1` and `orderstatus=4` (shipped, doesn't seem like there's a completed status).